### ISSUE TITLE
Installation error for onsenui@2.11.0

### ISSUE THREAD

**Issue by afroz1198** [OnsenUI/OnsenUI#2801]
```

> onsenui@2.11.0 preinstall /projects/challenge/node_modules/onsenui
> cd css-components && npm install
sh: 1: cd: can't cd to css-components
...
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! onsenui@2.11.0 preinstall: `cd css-components && npm install`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the onsenui@2.11.0 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```
:(

**Comment by afroz1198** [comment]
**package.json**
```
 "dependencies": {
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.3.2",
    "@testing-library/user-event": "^7.1.2",
    "onsenui": "^2.10.10",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-onsenui": "^1.11.2",
    "react-scripts": "3.4.1",
    "start-server-and-test": "^1.7.11",
    "wait-on": "^3.2.0"
}

```

**Comment by emccorson** [comment]
Thanks for alerting us to the issue.

I've now deprecated 2.11.0 and published 2.11.1. If you upgrade to 2.11.1, the install should be successful.
